WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/DashboardPostsListCardCell.swift

```diff
@@ -223,7 +223,5 @@ private extension DashboardPostsListCardCell {
 
     enum Constants {
         static let iconSize = CGSize(width: 18, height: 18)
-        static let constraintPriority = UILayoutPriority(999)
```

Comment:
@kean Curious to know why this constant had to be removed?

Reply:
Poor naming. It's clearer when it's used directly where it's needed.